roles/setup_dbt2/tasks/install_packages.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- name: Set the os variable
+  set_fact:
+    os: "{{ ansible_distribution | replace('RedHat', 'RHEL') }}{{ ansible_distribution_major_version }}"
+
+- name: Install packages for DBT-2 (yum)
+  yum:
+    name: "{{ dbt2_packages }}"
+  when: os in [ 'CentOS7', 'RHEL7']
+  become: yes
+
+- name: Install packages for DBT-2 (dnf)
+  dnf:
+    name: "{{ dbt2_packages }}"
+  when: os in [ 'CentOS8', 'RHEL8']
+  become: yes
```

Comment:
Perhaps you could use the `ansible.builtin.package` module for merging both cases into only one task

Reply:
Ok, consolidated.